src/app/view/render2D/featureRenderer2D.js

```diff
@@ -71,7 +71,8 @@ export function renderTextTarget(typeString, setString, position) {
     let rendered = new paper.PointText(new paper.Point(position[0], position[1]));
     rendered.justification = "center";
     rendered.fillColor = Colors.DEEP_PURPLE_500;
-    rendered.content = Registry.text;
```

Comment:
Since we are not getting text from the registry, we should get the text as an function argument.